ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json

```diff
@@ -128,12 +128,8 @@
           "target_property": "genetic_code"
         },
         {
-          "input_parameter": "generate_ids_if_needed",
```

Comment:
'generate_ids_if_needed' in 'parameters' and display.yaml should also be removed.